Tests/Tests/CDTSessionCookieInterceptorTests.m

```diff
@@ -49,18 +49,23 @@ - (void)setUp
       return [[request.URL host] isEqualToString:@"username.cloudant.com"];
     }
         withStubResponse:^OHHTTPStubsResponse *__nonnull(NSURLRequest *__nonnull request) {
-          return [OHHTTPStubsResponse
-              responseWithJSONObject:@{
-                  @"ok" : @(YES),
-                  @"name" : @"username",
-                  @"roles" : @[ @"_admin" ]
-              }
-                          statusCode:200
-                             headers:@{
-                                 @"Set-Cookie" :
-                                     [NSString stringWithFormat:@"%@; Version=1; Path=/; HttpOnly",
-                                                                testCookieHeaderValue]
-                             }];
+
+          if ([request.HTTPMethod isEqualToString:@"POST"]) {
+              return [OHHTTPStubsResponse
+                  responseWithJSONObject:@{
+                      @"ok" : @(YES),
+                      @"name" : @"username",
+                      @"roles" : @[ @"_admin" ]
+                  }
+                              statusCode:200
+                                 headers:@{
+                                     @"Set-Cookie" : [NSString
+                                         stringWithFormat:@"%@; Version=1; Path=/; HttpOnly",
+                                                          testCookieHeaderValue]
+                                 }];
+          } else {
+              return [OHHTTPStubsResponse responseWithJSONObject:@{} statusCode:200 headers:@{}];
```

Comment:
Couch returns `200` for a post to session see: http://docs.couchdb.org/docs/couchdb/en/stable/api/server/authn.html#session.

I can't check the body to make sure it is form encoded, so far my attempts to read HTTPBodyStream always fail. The only thing I can check is the content type